Assignment 2.Rmd

```diff
@@ -81,19 +93,41 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender)
 
 plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.width=EDGE$count)
 
-````
+```
 
 ## Part II
 
-In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
+# In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
+
+```{r}
+teacher_number <- EDGE %>% group_by (EDGE$from) %>% summarise (counts = sum(count))
+student_number <- EDGE %>% group_by (EDGE$to) %>% summarise (counts = sum(count))
+SIZE <- bind_rows(teacher_number, student_number)
```

Comment:
You can also create a new VERTEX dataframe by joining the VERTEX dataframe from Part 1 with the SIZE dataframe.